src/content/en/2020/http2.md

```diff
@@ -8,9 +7,9 @@ authors: [dotjs, rmarx, MikeBishop]
 reviewers: [LPardue, bazzadp, ibnesayeed]
 analysts: [gregorywolf]
```

Comment:
@gregorywolf is also listed as a reviewer in the [doc](https://docs.google.com/document/d/1ybIBEtM-C2e-pEZMs_bp39mtEs3u6s8LHOz6Es8QUGA/edit#heading=h.xleqpbggrlck). Is that correct? @dotjs

Reply:
Didn't get time to review and pinged me the other day to ask to be credited just as an analyst because of that. I told him @dotjs had already ruthlessly cut him from the reviewer credit 😉